plugins/Actions/tests/UI/ActionsDataTable_spec.js

```diff
@@ -13,68 +13,56 @@ describe("ActionsDataTable", function () {
     var url = "?module=Widgetize&action=iframe&idSite=1&period=year&date=2012-08-09&moduleToWidgetize=Actions&actionToWidgetize=getPageUrls&isFooterExpandedInDashboard=1";
 
     it("should load correctly", async function() {
-        expect.screenshot('initial').to.be.capture(function (page) {
-            page.load(url);
-        }, done);
+        await page.goto(url);
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('initial');
     });
 
     it("should sort column correctly when column header clicked", async function() {
-        expect.screenshot('column_sorted').to.be.capture(function (page) {
-            page.click('th#avg_time_on_page', 3000);
-        }, done);
+        await page.click('th#avg_time_on_page', 3000);
```

Comment:
FYI, the wait timeout no longer exists w/ chrome headless, instead you have to call one of the wait functions: https://github.com/GoogleChrome/puppeteer/blob/v1.4.0/docs/api.md#pagewaitforselectororfunctionortimeout-options-args

I added one extra one `waitForNetworkIdle()` that waits for network requests to finish. So here it would be:

```
await page.click('th#avg_time_on_page');
await page.waitForNetworkIdle();
```

Reply:
Also you can wait for times via `waitFor(1000)`, but have been trying to replace these w/ `waitFor('.selector')` unless its to wait out an animation. (Though in many cases they're unnecessary to have w/ chrome, and in other cases they can be reduced greatly since chrome is so much faster than phantomjs.)